src/Mod/Sketcher/Gui/ViewProviderSketch.cpp

```diff
@@ -385,7 +389,7 @@ void ViewProviderSketch::deactivateHandler()
         sketchHandler->unsetCursor();
         sketchHandler = nullptr;
     }
-
+    editCoinManager->setConstraintSelectability(true);

```

Comment:
Refer to comments on OpenBrain's PR.

Reply:
Right. I'll remove this commit from my branch when his PR merges and will rebase on master.